signature/README.md

```diff
@@ -0,0 +1,174 @@
+# Signature Module for Image-rs
+
+This is the signature module for image-rs. In fact, signature verification
+is included in the policy processing.
+
+## How is signature verification working?
+
+Up to now, all signature verification in image-rs happens due to
+the image security [policy](https://github.com/confidential-containers/image-rs/blob/main/docs/ccv1_image_security_design.md#policy) 
+file.
+
+The format of policy file is detailed [here](../docs/ccv1_image_security_design.md#policy).
+
+Each signing scheme works due to the `scheme` field in a [Policy Requirement](https://github.com/containers/image/blob/main/docs/containers-policy.json.5.md#policy-requirements).
+
+The format of a Policy Requirement may be little different from the mentioned link.
+
+A Policy Requirement claiming a specific signature scheme in the `scheme` field.
+Here are some examples for [Simple Signing](src/mechanism/simple/README.md)
+
+```json
+{
+    "type": "signedBy",
+    "scheme": "simple",
+    "keyType": "GPGKeys",
+    "keyPath": "/etc/pki/rpm-gpg/RPM-GPG-KEY-redhat-release",
+}
+```
+
+Here, the `type` field here must be `signedBy` if this Policy Requirement
```

Comment:
-> Here, the `signedBy` type shows that this Policy Requirement ...

Reply:
much clearer!